src/constants.ts

```diff
@@ -89,6 +91,18 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.MUMBAI]: "0x9c3C9283D3e44854697Cd22D3Faa240Cfb032889",
     },
   },
+  SNX: {
+    name: "Synthetix",
+    symbol: "SNX",
+    decimals: 18,
+    // Based on https://github.com/Synthetixio/synthetix-docs/blob/master/content/addresses.md
```

Comment:
Might want to peg the github link to a specific commit, in case future changes to addresses.md invalidate the statement.

```suggestion
    // Based on https://github.com/Synthetixio/synthetix-docs/blob/fe83d0757977b1cb7f69796126e71a66295bf264/content/addresses.md
```

Reply:
Yes, good point! Thanks 🙏